xchange-stream-kraken/src/main/java/info/bitrich/xchangestream/kraken/KrakenStreamingMarketDataService.java

```diff
@@ -143,7 +143,7 @@ public KrakenStreamingMarketDataService(KrakenStreamingService service) {
             });
   }
 
-  private String getChannelName(
+  public String getChannelName(
```

Comment:
Or this?

Reply:
same reason, I use these functions directly and would like a handle on them and receive updates for them if the underlying code changes.